src/workerd/api/r2-api.capnp

```diff
@@ -214,6 +216,10 @@ struct R2HeadResponse {
 
   checksums @8 :R2Checksums;
   # If set, the available checksums for this object
+
+  storageClass @9 :Text;
```

Comment:
non-blocking: Are these expected to remain fixed? Just wondering if an enum wouldn't be better. not critical tho

Reply:
We don't expect to remove any, but we might add more in the future, so a Text field makes that a little easier. I could see a case for validating the strings on the workerd side, though. For now I think we're ok with letting R2 interpret and validate the string.